gen/README.md

```diff
@@ -4,14 +4,16 @@ GDAL.jl generation documentation
 ### Install Clang.jl
 Because a few tricks are used to be able to properly write documentation,
 this fork of Clang.jl is used: https://github.com/visr/Clang.jl (branch gdal)
+For the generation process `julia 0.5-dev` based on `LLVM 3.7.1` and with
+`BUILD_LLVM_CLANG=1` in `Make.user` is used.
 
 ### Make sure you have the GDAL header files
-The `wrap_gdal.jl` script looks in `/usr/local/include` by default.
+The `wrap_gdal.jl` script finds them in the directory set in the `gdalpath` variable.
 
 ### Prepare the GDAL Doxygen XML
 1. In the GDAL SVN, run Doxygen with `GENERATE_XML = YES`
-2. In the `xml` output dir, run `combine.xslt` to combine all documentation in one XML
-3. Copy this file to `GDAL.jl/gen/all.xml`
+2. Follow the instructions in `combine_gdal_doxygen_xml.py`
+3. Copy the result to `GDAL.jl/gen/doxygen.xml`
```

Comment:
I've managed to follow all the steps, apart from manually modifying the xml file, and am getting errors like
```bash
$ julia gen/rewriter.jl
ERROR: LoadError: LoadError: "ErrorException(\"Error parsing document : 0\"), mismatched tag, 128156, 3, 8208979"
 in error at /Applications/Julia-0.4.2.app/Contents/Resources/julia/lib/julia/sys.dylib
 in xp_parse at /Users/yeesian/.julia/v0.4/LibExpat/src/LibExpat.jl:289
```
Is it okay host a copy of this generated file somewhere available in the meantime?